src/neptune/internal/utils/logger.py

```diff
@@ -49,12 +49,11 @@ def stream(self):
         return sys.stdout
 
 
-def get_logger(with_prefix: bool = True, loglevel: int = logging.DEBUG):
+def get_logger(with_prefix: bool = True):
```

Comment:
Does `_handle_fork_in_child` will still going to work as previously?

Reply:
Didn't notice that - seems that our unit tests don't cover it. That might be tricky

Reply:
@Raalsky good observation!